src/main/scala/org/constellation/consensus/Round.scala

```diff
@@ -16,7 +16,7 @@ import scala.collection.mutable
 import scala.concurrent.duration._
 import scala.concurrent.{ExecutionContextExecutor, Future}
 
-class Round(roundData: RoundData, dao: DAO) extends Actor with ActorLogging {
+class Round(roundData: RoundData, dao: DAO, dataResolver: DataResolver) extends Actor with ActorLogging {
```

Comment:
dataResolver could be mocked via PeerApiClient, why you didn't stick to that ?

Reply:
because I want to make unit test and mock only dependencies, not dependencies of dependencies